fedimint-core/src/encoding/mod.rs

```diff
@@ -575,6 +617,46 @@ impl Decodable for Cow<'static, str> {
     }
 }
 
+/// A writer counting number of writes written to it
+///
+/// Copy&pasted from https://github.com/SOF3/count-write which
```

Comment:
Clippy will warn you of:
```suggestion
/// Copy&pasted from <https://github.com/SOF3/count-write> which
```